<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSTWidgetUi</class>
 <widget class="QWidget" name="SSTWidgetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spatio-Temporal representation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color map:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AwComboCMap" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/AnyWave.qrc">
         <normaloff>:/images/back32x32.png</normaloff>:/images/back32x32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonForward">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/AnyWave.qrc">
         <normaloff>:/images/fwd32x32.png</normaloff>:/images/fwd32x32.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinDuration">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpdate">
       <property name="text">
        <string>Update Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPlots">
       <item>
        <property name="text">
         <string>Show Hilbert Transformation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show Amplitudes</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AwComboCMap</class>
   <extends>QComboBox</extends>
   <header location="global">widget/AwComboCMap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
 </resources>
 <connections/>
</ui>
